### ISSUE TITLE
designsystem.digital.gov a11y: Header

### ISSUE THREAD

**Issue by jAigret-Bix** [uswds/uswds-site#1094]
## Description
Related to PR #1092  a11y issues found re: `Header`
Related to https://github.com/uswds/uswds-site/issues/1093 `SCOPE`
Related to uswds/uswds/issues/3733

<details><summary>Page templates - 1 axe violation</summary>

```bash
/page-templates/ Found 1 aXe violations.
[
  {
    id: 'heading-order',
    impact: 'moderate',
    tags: [ 'cat.semantics', 'best-practice' ],
    description: 'Ensures the order of headings is semantically correct',
    help: 'Heading levels should only increase by one',
    helpUrl: 'https://dequeuniversity.com/rules/axe/3.5/heading-order?application=axeAPI',
    nodes: [ [Object], [Object] ]
  }
]
```
</details>

⚠️ These issues will also appear on the individual Component pages once we start testing those. 

## Related Issues
- [ USWDS - Testing: Add ability to test subpages ie Component subpages. #1090](https://github.com/uswds/uswds-site/issues/1090)
- [ USWDS - Components: Create yml file for component variants. #1091](https://github.com/uswds/uswds-site/issues/1091)
